src/watchmaker/workers/salt.py

```diff
@@ -986,6 +998,7 @@ def install(self):
         if self.pip_install:
             self._install_pip_packages()
         self.salt_call = SaltWindows._get_salt_call()
+        self.log.info("Salt minion installation completed successfully.")
```

Comment:
I think we can remove this log line, since we're adding a check for the salt version, and might not have installed salt at all. That means this line would be misleading.

```suggestion
```